docs/src/understand/index.md

```diff
@@ -1,34 +1,87 @@
 
-2. The MQTT bus is connected to the cloud, forwarding the messages published on cloud specific topics. 
+A typical thin-edge setup consists of the following components:
+- a local MQTT broker that is used as a message bus between all the components of the equipment
+- an MQTT bridge connection between the local message bus and the remote cloud end-point
+- thin-edge (out-of-the-box) device management services which provide features such as monitoring, configuration and software management
+- equipment-specific services that interact with the hardware that make the equipment,
+  publishing the collected data and forwarding operation requests on the MQTT bus as well as abstracting various device specific protocols
+- a cloud-specific service that maps the messages exchanged on the local bus with messages sent to or received from the cloud
 
-2. A [canonical data format](thin-edge-json.md) let the components exchange telemetry data independently of the connected cloud.
-This is an optional feature, and the components are free to also use cloud specific data formats.
+The first point to note is that all these software components can be provided by independent vendors:
+by the thin-edge open-source project, by the equipment maker, by the IoT application developer
+or even by cloud providers, hardware manufacturers and protocol implementors. 
+Their interoperability is based on:
+- ubiquitous protocols: JSON over MQTT and HTTP,
+- dynamic and loose inter-process communication with no constraint on programming tools nor software placement,
+- domain-specific APIs that can be used independently:
+  for measurements, events, alarms, configurations, software updates, firmware updates,   
+- various extension points: for child devices, services, operations and clouds.
 
-3. The mapper processes are responsible for translating the canonical data format into cloud specific messages and vice versa.
+The second key point is that thin-edge not only defines a set of APIs
+but also provides a set of ready-to-use software components implementing the associated features.
+Out-of-the-box thin-edge supports telemetry and device management features on the main devices and child devices.
+These features are implemented by composable software components that:
+- can be freely adapted, combined, removed or replaced
+- provide the foundation to start the development of an agent with sensible defaults
+- on top of which the specificities required by smart IoT agents can be incrementally implemented
 
+## Why
 
-![Overview](./images/thin-edge-overview.png)
+The aim of thin-edge is to reduce the development effort to build smart IoT agents
+without compromising on quality and feature completeness.
 
+- ready-to-use components provide sound foundations
+- interchangeable software components make it possible to adapt the generic agent to specific contexts
+- simple, yet flexible, extension points enable custom functionality to be added in a modular fashion
```

Comment:
These bullet points are missing one sentence to put the statements into more context.

Something like:

```
thin-edge's goal is to deliver:
```